lib/tasks/braintree_tokens.rake

```diff
@@ -0,0 +1,8 @@
+namespace :champaign do
+  desc "Create braintree payment method tokens out of existing token columns"
+  task braintree_tokens: :environment do
+    Payment::BraintreeCustomer.all.each do |customer|
+      Payment::BraintreePaymentMethod.create!({customer_id: customer.customer_id, token: customer.card_vault_token})
```

Comment:
We changed how that works - there's no field for it now, there's a method in the customer model to get `default_payment_method`, which is the most recently used payment method. Omar's idea was to not have this column and to instead use a boolean for `default` in the payment methods table if we'll eventually want to change the logic on what's the default one.